### PR TITLE
elogind: update to 241.2.

### PR DESCRIPTION
**Pull request by Cogitri**
Not quite sure how to test if elogind actually doesn't kill the user process
upon logging out

### CODE REVIEW

**Comment by Cogitri** [comment]
I think @Gottox might now that?

**Comment by Gottox** [comment]
What do you mean with "test"?

**Comment by Cogitri** [comment]
Well, making sure that processed in fact aren't killed when the user logs out. Does making a `screen` in a Terminal and relogging and attaching back to it suffice for testing?

File: srcpkgs/elogind/template [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-version=241.1
-revision=2
+version=241.2
+revision=1
 build_style=meson
 configure_args="-Dcgroup-controller=elogind -Dhalt-path=/usr/bin/halt
  -Drootlibexecdir=/usr/libexec/elogind -Dreboot-path=/usr/bin/reboot
- -Dkexec-path=/usr/bin/kexec -Ddefault-hierarchy=legacy"
+ -Dkexec-path=/usr/bin/kexec -Ddefault-hierarchy=legacy
+ -Ddefault-kill-user-processes=false"
```
Comment (maxice8) [inline srcpkgs/elogind/template]: @Vaelatern can test ?

**Comment by Gottox** [comment]
what's the problem? `sed -i '/KillUserProcesses=/s/.*/KillUserProcesses=No/' logind.conf` isn't enough here?

**Comment by Cogitri** [comment]
No, upstream offers a meson configure option now, so we don't have to sed it anymore. I'd just like you to confirm that upstream's way works, since the KillUserProcesses=No is commented out.

**Comment by Vaelatern** [comment]
Have to agree with Gottox. Why is a changed compile option required?

**Comment by Cogitri** [comment]
Well, I figured that if upstream offers an option for that we might as well use it instead of sedding the finished config file which seems like less work our side.

**Comment by Cogitri** [comment]
Alright, at least screen and tmux sessions survive a logout, so I guess this is fine.

**Pull request merged**